packages/block-editor/src/components/rich-text/index.js

```diff
@@ -327,6 +324,29 @@ class RichTextWrapper extends Component {
 		};
 	}
 
+	renderFormatToolbar( inline ) {
+		if ( inline && Platform.OS === 'web ' ) {
```

Comment:
Is there any way we can split this in a separate file and use the current `.native` system?